Python/ibmcloudsql/test.py

```diff
@@ -3,7 +3,7 @@
 import ibmcloudsql
 import test_credentials
 import pandas as pd
-pd.set_option('display.max_colwidth', -1)
+pd.set_option('display.max_colwidth', None)
 pd.set_option('display.max_columns', 20)
 
```

Comment:
Please can you add a series of system test steps here for all the new externalised methods that you introduced in SQLQuery.py?

Reply:
The tests are currently integrated in each module. I will move them to the test.py file.

Reply:
It's fine to keep some tests in the __main__ method. But what you have there is not complete coverage anyway yet. I always run the test.py e2e as the key gatekeeper before releasing new library version on pypi.

Reply:
Yes, I added them now in both test.py and ./test/ folders (pytest).